migrations/1659106715329-Deposit.ts

```diff
@@ -0,0 +1,38 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Deposit1659106715329 implements MigrationInterface {
+  name = "Deposit1659106715329";
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DROP VIEW IF EXISTS "deposit_referral_stats";`);
+    await queryRunner.query(`
+      CREATE VIEW "deposit_referral_stats" as
+      (
+        select
+          d1.id,
+          count(distinct d2."depositorAddr") as "referralCount",
+          sum((d2.amount / power(10, t.decimals)) * hmp.usd) as "referralVolume"
+        from deposit d1
+        left join deposit d2 on d1."stickyReferralAddress" = d2."stickyReferralAddress"
```

Comment:
this is the same old view, only that `stickyReferralAddress` replaces `referralAddress` column